jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/UriCompliance.java

```diff
@@ -192,13 +197,42 @@ public String getDescription()
     public static final UriCompliance DEFAULT = new UriCompliance("DEFAULT", RFC3986.getAllowed());
 
     /**
-     * LEGACY compliance mode that models pre Jetty 12 behavior by allowing:
+     * JETTY_11 compliance mode that models Jetty 11 behavior by allowing:
+     * <ul>
+     *     <li>{@link Violation#AMBIGUOUS_PATH_SEGMENT}</li>
+     *     <li>{@link Violation#AMBIGUOUS_PATH_SEPARATOR}</li>
+     *     <li>{@link Violation#AMBIGUOUS_PATH_ENCODING}</li>
+     *     <li>{@link Violation#BAD_UTF8_ENCODING}</li>
+     *     <li>{@link Violation#SUSPICIOUS_PATH_CHARACTERS}</li>
+     *     <li>{@link Violation#UTF16_ENCODINGS}</li>
+     *     <li>{@link Violation#USER_INFO}</li>
+     * </ul>
+     * <p>
+     *     Note: this mode allows URL/URIs that the Servlet spec will reject.
+     *     <br>
+     *     See <a href="https://github.com/jakartaee/servlet/blob/6.0.0-RELEASE/spec/src/main/asciidoc/servlet-spec-body.adoc#352-uri-path-canonicalization">point 10 "Rejecting Suspicious Sequences" in Section 3.5.2. URI Path Canonicalization</a>,
+     *     <br>
+     *     and <a href="https://jetty.org/docs/jetty/12/programming-guide/server/compliance.html#servleturi">Jetty Documentation: Servlet URI Compliance Modes.</a>
+     * </p>
+     */
+    public static final UriCompliance LEGACY_11 = new UriCompliance("LEGACY_11",
```

Comment:
so the LEGACY will not include `Violation.SUSPICIOUS_PATH_CHARACTERS`?

Reply:
Lets call this `JETTY_11`, and update javadoc to say that it models the Jetty 10/11 DEFAULT setting.

Reply:
> so the LEGACY will not include `Violation.SUSPICIOUS_PATH_CHARACTERS`?

LEGACY is really LEGACY LEGACY (i.e what jetty-11 does when in LEGACY mode).   That's what the testing revealed!